src/schema/broadband-signup-au.json

```diff
@@ -141,28 +181,172 @@
             "properties": {
                 "name": {
                     "type": "string",
-                    "example": "Monthly pay as you go",
+                    "example": "Internet 100",
                     "minLength": 1
                 },
-                "priceLine": {
+                "description": {
+                    "$ref": "#/domains/Generic/types/StructuredTextItem"
+                },
+                "price": {
+                    "$ref": "#/domains/Generic/types/Price"
+                }
+            },
+            "required": [
+                "name"
+            ],
+            "additionalProperties": false
+        },
+        "AddressStatus": {
+            "type": "object",
+            "properties": {
+                "currentAddress": {
+                    "type": "object",
+                    "properties": {
+                        "line1": {
+                            "type": "string",
+                            "example": "50 CRESTHAVEN BVD",
+                            "minLength": 1
+                        },
+                        "line2": {
+                            "type": "string",
+                            "example": "Flat 2"
+                        },
+                        "postCode": {
+                            "type": "string",
+                            "example": "3806",
+                            "minLength": 1
+                        },
+                        "city": {
+                            "type": "string",
+                            "example": "Melbourne",
+                            "minLength": 1
+                        },
+                        "citySubdivision": {
+                            "type": "string",
+                            "example": "Berwick",
+                            "minLength": 1
+                        },
+                        "countrySubdivision": {
+                            "$ref": "#/domains/BroadbandSignupAU/types/StateCode"
+                        },
+                        "status": {
+                            "type": "string",
+                            "enum": [
+                                "owned",
+                                "rent",
+                                "others"
+                            ]
+                        },
+                        "moveingInDate": {
+                            "type": "string",
+                            "format": "date",
+                            "example": "2001-01-01"
+                        }
+                    },
+                    "required": [
+                        "line1",
+                        "postCode",
+                        "city",
+                        "citySubdivision",
+                        "countrySubdivision",
+                        "status"
+                    ],
+                    "additionalProperties": false
+                },
+                "previousAddress": {
```

Comment:
Can there only be 1 previous address?

I believe `currentAddress` and `previousAddresses` should be separate inputs, because they may be required at different points during the flow.

Reply:
They are requested in the same place always. I don't think there is more than one previous address.